### DISCUSSION TITLE
How to use 'yaml' or 'json' file in dynamic-prompts A1111 extension?

### DISCUSSION THREAD

**Discussion by 925-Studio** [adieyal/sd-dynamic-prompts#428] [Q&A]
I know it's a silly question but I've looked almost everywhere but still can't figure it out. All the 'txt' files in 'wildcards' folder are working fine. I put 'yaml' and 'json' files into the same folder, but they just won't appear.

**Comment by 925-Studio** [comment]
I just found out, the file should be in the `.../wildcards/`, can't place them in `.../wildcards/subfolder`.

**Reply by adieyal** [reply]
@akx I think I agree with @925-Studio, is there a reason why we shouldn't allow yaml and json files to be placed in an arbitrary location in the wildcards directory?

**Reply by akx** [reply]
@adieyal Um, we do already allow that. See e.g https://github.com/adieyal/dynamicprompts/blob/main/tests/test_data/wildcards/flavors/pantry.json (which expands to `wildcards/flavors/bitter`)
